src/pymorize/cli.py

```diff
@@ -149,7 +151,11 @@ def cache(verbose, quiet, logfile, profile_mem):
     return 0
 
 
-################################################################################
+@click.group()
+def scripts():
+    return 0
+
+
```

Comment:
It seems that this is not used in the end, can it be cleaned up?

Reply:
This is a bit deceptive! I added this to allow users to run specific steps in a standalone fashion without firing up the whole machinery (request by @chrisdane). So, rather than defining the entire pipeline stuff, if you want to run this particular step (fesom1 3D to 2D (node vs level)) you can do that from the command line interface:

```
$ pymorize scripts fesom1 nodes-to-levels <arguments>
```

Reply:
I see